pkg/components/minio/minio.go

```diff
@@ -24,34 +25,92 @@ func Instance(mattermost *mattermostv1alpha1.ClusterInstallation) *minioOperator
 		minioName = mattermost.Spec.Minio.Secret
 	}
 
+	return newMinioInstance(
+		minioName,
+		mattermost.Namespace,
+		mattermostv1alpha1.ClusterInstallationResourceLabels(mattermost.Name),
+		mattermostApp.ClusterInstallationOwnerReference(mattermost),
+		mattermost.Spec.Minio.Replicas,
+		mattermost.Spec.Minio.StorageSize,
+	)
+}
+
+// Secret returns the secret name created to use togehter with Minio deployment
```

Comment:
```suggestion
// Secret returns the secret name created to use together with Minio deployment
```

Reply:
Just checking, this is for the alpha CR and the other one below is for beta, right?

Reply:
Yes, that is correct